### PR TITLE
Verify User - UI

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds UI elements for functionality in https://github.com/ConnorDY/collective-shield/pull/88

Expected flow:
- Login as printer
- Get redirected to agreement form
- Fill out the form and submit
- See status message
- Log out
- Log back in as printer
- See message saying that your request is awaiting approval
- Log out
- Set `isVerifiedMaker` on the database
- Log back in as printer
- You should see the Work screen

Form:
Screen Shot 2020-04-09 at 7 02 26 PM

Confirmation:
Screen Shot 2020-04-09 at 7 02 43 PM

Awaiting approval:
Screen Shot 2020-04-09 at 7 03 00 PM

### CODE REVIEW

File: ui/src/views/MakerVerificationPendingView.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {
+  Alert,
+} from 'react-bootstrap';
+
+const VerificationPendingView: React.FC<{}> = () => {
+  return (
+    <div className="c-requestForm -pad">
+      <Alert variant="info">
+        {`If you have been re-directed to this page, it is because your Maker account is currently under review.
```
[the remark this answers is not in the document]
Comment (mjoyce91) [reply ui/src/views/MakerVerificationPendingView.tsx:10]: Line breaks make it easier for a developer to read

File: ui/src/views/MakerVerificationView.tsx [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+          </h1>
+        </Col>
+      </Row>
+
+      {isCreated ? (
+        <Row>
+          <Col>
+            <div className="c-requestForm -pad">
+              <Alert variant="success">
+                {' '}
```
[the remark this answers is not in the document]
Comment (mjoyce91) [reply ui/src/views/MakerVerificationView.tsx:118]: Probably not, copy pasta

File: ui/src/views/MakerVerificationPendingView.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (vadnguyen) [reply ui/src/views/MakerVerificationPendingView.tsx:10]: we should really provide a link back to home page/squarespace site or explain that they can close out of the tab. Better UX than just abandoning user.

[the remark this answers is not in the document]
Comment (vadnguyen) [reply ui/src/views/MakerVerificationPendingView.tsx:10]: +1 on why in template string ^

[the remark this answers is not in the document]
Comment (mjoyce91) [reply ui/src/views/MakerVerificationPendingView.tsx:10]: @vadnguyen - I think that is currently being addressed by making the logo navigate the user back to collectiveshield.org

[the remark this answers is not in the document]
Comment (vadnguyen) [reply ui/src/views/MakerVerificationPendingView.tsx:10]: @mjoyce91 yea, I recall seeing that, but should have a link in addition to in that message. It's not obvious to the user at the moment.

[the remark this answers is not in the document]
Comment (ConnorDY) [reply ui/src/views/MakerVerificationPendingView.tsx:10]: Prettier should automatically add line breaks to strings like this when they're not in a template string. If it doesn't you can always add this above the line:
```typescript
// prettier-ignore
```

[the remark this answers is not in the document]
Comment (mjoyce91) [reply ui/src/views/MakerVerificationPendingView.tsx:10]: I don't actually want a line break in the rendered site though

[the remark this answers is not in the document]
Comment (ConnorDY) [reply ui/src/views/MakerVerificationPendingView.tsx:10]: There won't be a rendered line break unless you add a `<br />` or wrap it in `<pre>` tags no matter how many like breaks you add in the JSX.

[the remark this answers is not in the document]
Comment (mjoyce91) [reply ui/src/views/MakerVerificationPendingView.tsx]: Guess I didn't even need the prettier ignore

**Pull request merged**